src/index.html

```diff
@@ -1,12 +1,348 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="/images/logo.png" alt="logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="products">
+          <a href="#" class="products__link is-active">Apple</a>
+          <a href="#" class="products__link">Samsung</a>
+          <a href="#" class="products__link">Smartphones</a>
+          <a
+            href="#" class="products__link"
+            data-qa="nav-hover"
+          >
```

Comment:
Fix all similar cases
```suggestion
          <a
            href="#"
            class="products__link"
            data-qa="nav-hover"
          >
```